### PR TITLE
lightdm 2.0 fixes.

### PR DESCRIPTION
**Pull request by wavexx**
- Correctly set menu colors for the top bar.
- Fix default login button color.
- Set correct warning color on invalid password.

### CODE REVIEW

File: gtk-3.0/apps/lightdm-gtk-greeter.css [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
     background: transparent;
+    background-image: none;
 }
 
 /*********
  * panel *
  *********/
 #panel_window {
-    background-color: transparent;
+    background-color: @menu_bg_color;
```
Comment (satya164) [inline gtk-3.0/apps/lightdm-gtk-greeter.css]: Making the top bar transparent is a design decision we took.

Comment (wavexx) [inline gtk-3.0/apps/lightdm-gtk-greeter.css]: On 05/23/2015 07:42 PM, Satyajit Sahoo wrote:

> Making the top bar transparent is a design decision we took.

Fair enough, what about the rest?

**Comment by satya164** [comment]
Rest are fine. Just remove the panel changes and push before we can merge.

**Comment by wavexx** [comment]
Updated.

**Pull request merged**